### ISSUE TITLE
Задание 1. Иван, дополните инструкцию

### ISSUE THREAD

**Issue by a-grinyuk** [oknosoft/metadata-devtraining#18]
Раздел 6. База 1С
Нужно включить CORS, All domain

Раздел 7. Запуск приложения
Меняем не только couch_path:
Но и couch_local:

**Comment by wecansu** [comment]
Согласен, надо дополнить, у меня так и не удалось его победить (CORS)

**Comment by DBMainUser** [comment]
Плюсую, здесь писал уже про это https://github.com/oknosoft/metadata-devtraining/issues/16

**Comment by nmivan** [comment]
Ребята, сорян что не отвечаю, разгребу неотложку и двинемся.

**Comment by unpete** [comment]
Пока Иван не отвечает, подумайте над фразой из https://github.com/oknosoft/metadata-devtraining/issues/18#issue-296177705
`couch_local` нужна, чтобы пересобрать метаданные. В работе web-приложения вообще не используется. Никак.
`couch_path` задаёт умолчание. Его можно изменить на время разработки, а можно и не менять, записав нужный вам путь в `localStorage`, но в production-режиме, взаимодействие с Couchdb надо проксировать в основной origin (домен + порт) приложения. Именно это и предлагается в настройках по умолчанию: `couch_path = '\couchdb\'`

**Comment by a-grinyuk** [comment]
> couch_local нужна, чтобы пересобрать метаданные. В работе web-приложения вообще не используется. Никак.
Вот когда я пересобирал метаданные, вот тогда и выдавалась ошибка. Нашёл, исправил. Вот тогда "meta:prebuild" выполнился полностью.

**Comment by wecansu** [comment]
Как все оказалось просто, я всего то один флажок, по описанию "Раздел 6. База 1С
Нужно включить CORS, All domain" вообще не понял что надо, и прописывал в разделе http и чего только не делал, а всего то надо было поставить флажок. Картинку прикрепляю.
![2018-02-20 17 25 16](https://user-images.githubusercontent.com/35488227/36429282-5882dd8e-1663-11e8-8b7b-245cb773e288.png)
